### ISSUE TITLE
SkeletonIK system in version 3.3 is not working correctly.

### ISSUE THREAD

**Issue by okay300** [godotengine/godot#48358]
**1.SkeletonIK system in version 3.3 is not working correctly.**
![IK_____________1](https://user-images.githubusercontent.com/75214342/116794483-ab6a4300-aae6-11eb-8979-cf865b1f3885.png)

**2. Animation works wrong scale.**                   Is it possible to get around this?
![Scale________1](https://user-images.githubusercontent.com/75214342/116794487-b1f8ba80-aae6-11eb-972c-211b9ed9569c.png)

**Godot and blender project:**
[Ik_test.zip](https://github.com/godotengine/godot/files/6410270/Ik_test.zip)

**Comment by Calinou** [comment]
Duplicate of https://github.com/godotengine/godot/issues/48078.

**Comment by akien-mga** [comment]
Reopening as discussed in #48078, this was not a duplicate (or only a partial one, there's still a separate issue after #48078 was fixed).

See https://github.com/godotengine/godot/issues/48078#issuecomment-833992369

![](https://user-images.githubusercontent.com/75214342/117383658-b237f200-aefa-11eb-84a7-14c8e9e730d6.png)

Reproduction project: [godot and blender project](https://github.com/godotengine/godot/files/6438387/godo.and.blender.project.zip)
